docs/src/main/sphinx/installation/cli.rst

```diff
@@ -77,9 +77,18 @@ The detailed behavior is as follows:
   the query.
 * Further queries in the CLI session do not require additional logins while the
   authentication token remains valid. Token expiration depends on the external
-  authentication system configuration.
+  authentication type configuration.
```

Comment:
The original was correct. Token expiration is controlled by the SSO provider they are using.

Reply:
OK